internal/discord/commands.go

```diff
@@ -0,0 +1,5 @@
+package discord
```

Comment:
Make sure you use the [authentication client](github.com/Floor-Gang/authclient) to validate each user using a command. If you need help setting this up, let me know.